l10n_es_sigaus_invoice/models/product_category.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    sigaus_subject = fields.Boolean(string="Subject to SIGAUS")
```

Comment:
```suggestion
    sigaus_subjected = fields.Boolean(string="Subjected to SIGAUS")
```

Reply:
En este caso, la forma correcta en inglés es "subject to" y no " subjecte to". Puedes consultarlo aquí: https://www.wordreference.com/es/en/translation.asp?spen=sujeto

Reply:
Ya veo. My fault...